src/api-v1.js

```diff
@@ -658,13 +659,17 @@ api.declare({
       token: token,
     });
 
+    let workerId = slugid.v4();
+    let workerGroup = 'ec2_' + secret.region;
```

Comment:
I think the workerGroup should be `aws-provisioner` (maybe with `-v1`).  Without the instance ID, the region doesn't make a lot of sense.